src/main/java/Tractor.java

```diff
@@ -0,0 +1,27 @@
+public class Tractor implements Drivable, Tradable{
+    private double maxSpeed;
```

Comment:
Consider changing the type of maxSpeed to int since we seem to only be working with integer speeds

Reply:
fixed!